include/grpc/impl/codegen/grpc_types.h

```diff
@@ -216,7 +216,7 @@ typedef struct {
 #define GRPC_ARG_HTTP2_WRITE_BUFFER_SIZE "grpc.http2.write_buffer_size"
 /** Should we allow receipt of true-binary data on http2 connections?
     Defaults to on (1) */
-#define GRPC_ARG_HTTP2_ENABLE_TRUE_BINARY "grpc.http2.true_binary"
```

Comment:
From an API perspective, are we allowed to do this ?

Reply:
Change an existing channel arg? Yes, but it will require a gRFC and a core version bump.

Quick search of github and google3 shows that this is not consumed by anyone